python/timeplus/api.py

```diff
@@ -0,0 +1,159 @@
+import json
```

Comment:
`api` is too generic, shall we call it `dbAPI` or something else? Maybe I'm not very familiar with this, I think `dbAPI` is too generic as well.

Reply:
Also, shall we put it into `superpartner` repo? IMO this is a very PoC code, we probably should not put it into SDK (which is supposed to be stable)

Reply:
> Also, shall we put it into `superpartner` repo? IMO this is a very PoC code, we probably should not put it into SDK (which is supposed to be stable)

I found some other db libs put them together, so it should be fine here